### PR TITLE
Add Porgverse

### PR DESCRIPTION
**Pull request by Jazzkuh**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress your images.

**Comment by Jazzkuh** [comment]
> Please compress your images.

I did compress them

**Comment by JakobBruening** [comment]
Really? I can still improve the file size: https://compresspng.com/

**Comment by Jazzkuh** [comment]
> Really? I can still improve the file size: https://compresspng.com/

Oh ill take a look at it tomorrow then!

**Comment by Jazzkuh** [comment]
> > Really? I can still improve the file size: https://compresspng.com/
> 
> Oh ill take a look at it tomorrow then!

Done

File: minecraft_servers/porgverse/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+     "primary": "#ff9900",
+     "background": "#ad6b08",
+     "text": "#FFFFFF"
+  },
+  "supported_languages": [
+    "nl"
+  ],
+  "location": {
+    "country": "The Netherlands",
+    "country_code": "nl"
```
Comment (JakobBruening) [inline minecraft_servers/porgverse/manifest.json:15]: Please use uppercase here.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
- background-min.png
- background@2x-min.png
- icon-min.png
- icon@2x-min.png

The file names are not supported. [Documentation reference](https://github.com/LabyMod/server-media/blob/master/docs/Files.md)

BR, Jakob

File: minecraft_servers/porgverse/manifest.json (hunk 1 above)
Comment (Jazzkuh) [reply minecraft_servers/porgverse/manifest.json]: Done

**Review by JakobBruening — APPROVED** [review]
Looks good now, thank you!

**Pull request merged**